src/tmpfiles/tmpfiles.c

```diff
@@ -617,8 +617,20 @@ static int path_set_perms(Item *i, const char *path) {
          * O_PATH. */
 
         fd = open(path, O_NOFOLLOW|O_CLOEXEC|O_PATH);
-        if (fd < 0)
-                return log_error_errno(errno, "Adjusting owner and mode for %s failed: %m", path);
+        if (fd < 0) {
+                int level = LOG_ERR, r = -errno;
+
+                /* Option "e" operates only on existing objects. Do not
+                 * print errors about non-existent files or directories */
+                if (i->type == EMPTY_DIRECTORY && errno == ENOENT) {
+                        level = LOG_DEBUG;
+                        r = 0;
+                }
+
+                log_full(level, "Adjusting owner and mode for %s failed: %m", path);
+
+                return r;
```

Comment:
oh, i'd really always return the error code properly, and make the caller ignore it if it desires so... but i guess your version works too and has the benefit that we don't need two checks for the same issue. 

One small nitpick, please use log_full_errno() instead of log_full(). That way the error number is included in the structured log data if we log to the journal.